test/util/providers/common.go

```diff
@@ -111,6 +119,10 @@ func DeleteObjectsInBucket(cloudProvider, cloudCredentialsFile, bslBucket, bslPr
 
 func SnapshotsShouldNotExistInCloud(cloudProvider, cloudCredentialsFile, bslBucket, bslConfig, backupName string, snapshotCheckPoint SnapshotCheckPoint) error {
 	fmt.Printf("|| VERIFICATION || - Snapshots should not exist in cloud, backup %s\n", backupName)
+	if cloudCredentialsFile == "" {
+		fmt.Printf("|| SKIPPED || - Skipping snapshots checkpoint %s\n", backupName)
+		return nil
+	}
```

Comment:
I think it's better to throw error currently since credential is required as test utils implemented, in future if cacert is support by test utils, we can update this part then.